crates/node/core/src/node_config.rs

```diff
@@ -45,6 +45,9 @@ pub const DEFAULT_PERSISTENCE_THRESHOLD: u64 = 2;
 /// Default size of cross-block cache in megabytes.
 pub const DEFAULT_CROSS_BLOCK_CACHE_SIZE_MB: u64 = 4 * 1024;
 
+/// Default parallelism threshold for the parallel sparse trie.
+pub const DEFAULT_PARALLEL_SPARSE_TRIE_THRESHOLD: usize = 100;
```

Comment:
can we put this constant in a place where it makes sense + can be shared by the `node/` code and `engine/` code?